studio/style.css

```diff
@@ -273,6 +273,26 @@ merch-card .overlay {
     height: 100%;
 }
 
+merch-card[variant='plans'][size='wide'] {
```

Comment:
can you move merch-card styles in merch-card.css?  I think it is time to do it.

Reply:
I added `merch-card {width: px}` for wide and super-wide cards. I had to do this here because in mas studio we don't use grids so the wide cards cannot span across multiple columns since there are no columns. So I had to simulate wide and super-wide cards with fixed widths. Since we need this only in mas studio I still don't think we should move it to merch-card.css
We discussed about this in fishbags and I think @npeltier said that all this will be redesigned in mas studio and for now I can simply set these fixed widths.

Reply:
what I meant is that, we can have merch-card.css in mas studio, since style.css is growing big and maybe it would be easier to keep merch card stying for MAS studio in a separate file.

Reply:
ahaaaa :) done